R/pair_count.R

```diff
@@ -6,21 +6,65 @@ pair_count <- function (df,
                         distinct = TRUE,
                         diag = FALSE,
                         sort = TRUE,
-                        unite = FALSE){
+                        unite = FALSE,
+                        group_by = NULL){
+  if(!is.null(group_by)) {
+    df <- df %>% dplyr::group_by(!!!rlang::sym(group_by))
```

Comment:
Should be !! rather than !!!

Reply:
Here is how to take column without quotation.
group_by_col <- tidyselect::vars_select(names(df), !! rlang::enquo(group_by))
df <- df %>% dplyr::group_by(!!!rlang::sym(group_by_col))

Reply:
Changed to use `group_by_col <- col_name(substitute(group_by))` to align with other arguments.